serveradmin/settings.example.py

```diff
@@ -15,11 +15,13 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'HOST': 'localhost',
         'NAME': 'serveradmin',
+        'USER': 'serveradmin',
+        'PASSWORD': 'serveradmin',
         'OPTIONS': {
             'connect_timeout': 1,
             'client_encoding': 'UTF8',
-            'options': '-c lock_timeout=60000',
```

Comment:
I agree this setting is not a very good idea, though I though it was before.  It would be nice to remove this with a separate commit.

Reply:
Fixed